components/org.wso2.carbon.stream.processor.core/src/main/java/org/wso2/carbon/stream/processor/core/event/queue/EventListMapManager.java

```diff
@@ -152,7 +171,10 @@ public void trimAndSendToInputHandler() throws InterruptedException {
                     for (Source source : sources) {
                         if(queuedEvent.getSourceHandlerElementId().equals(source.getMapper().
                                 getHandler().getElementId())){
-                            source.getMapper().getHandler().sendEvent(queuedEvent.getEvent());
+                            source.getMapper().getHandler().sendEvent(queuedEvent.getEvent(),
+                                    null);
```

Comment:
Why are you not using 
sendEvent(queuedEvent.getEvent()) 
and passing null ?

Reply:
@suhothayan sorry did not get the question . Are you asking why sending the null without sending queuedEvent.getTransportSyncProperties() ?